app/styles/play/campaign-view.sass

```diff
@@ -376,7 +376,7 @@ $gameControlMargin: 30px
         height: 32px
         color: white
 
-      .gem, .player-level-icon, .player-hero-icon
+      .gem, .level-indicator .player-level-icon, .player-hero-icon
```

Comment:
Doesn't it need a comma after .level-indicator, or doesn't it make any difference?